Generator/Sources/NeedleFramework/Generating/Pluginized/PluginizedDependencyProviderSerializerTask.swift

```diff
@@ -39,10 +40,10 @@ class PluginizedDependencyProviderSerializerTask: AbstractTask<[SerializedProvid
         // Group the providers based on where the properties are coming from
         // This will allow us to extract common code for multiple depndency providers
         // into common base classes
-        var counts = [[PluginizedProcessedProperty]: [PluginizedProcessedDependencyProvider]]()
+        var counts = OrderedDictionary<[PluginizedProcessedProperty], [PluginizedProcessedDependencyProvider]>()
```

Comment:
What's the perf difference with this change?

Reply:
No visible perf impact.
After the change, 344ms (0.7% of total time) is spent in OrderedDictionary related operations.